src/Behat/Gherkin/Loader/CucumberNDJsonAstLoader.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+namespace Behat\Gherkin\Loader;
```

Comment:
This is in the main code rather than a test folder because

a. it follows where the Yaml loader is
b. it may be useful if we ever want to outsource the parser to the gherkin binary